src/scss/06_components/paragraphs/_service-list.scss

```diff
@@ -8,9 +8,18 @@
   }
 
   .views-exposed-form {
+
+    padding-bottom: 30px;
+    
+    border-bottom: 1px solid $color-black-10;
+    @include breakpoint($breakpoint-l) {
+      padding-bottom: 72px;
```

Comment:
in general, check if there are common variables for these paddings / margins etc.. 72 seems too precise and it might miss the standards from wesite